src/Roave/NonceUtility/Service/NonceService.php

```diff
@@ -76,18 +76,20 @@ public function __construct(ObjectManager $objectManager, NonceRepositoryInterfa
     }
 
     /**
-     * {@Inheritdoc}
+     * Creates and persists a nonce entity
+     *
+     * @param $nonce
+     * @param NonceOwnerInterface|null $owner
+     * @param DateInterval|null $expiresIn
+     * @param string $namespace
+     * @return NonceEntity
      */
-    public function createNonce(
-        NonceOwnerInterface $owner,
-        $namespace = 'default',
+    private function _create(
```

Comment:
Please don't prefix method names. Also, removing public methods is a bc break too